AppDB/dbconstants.py

```diff
@@ -33,6 +33,7 @@
 APP_ENTITY_TABLE = "ENTITIES__"
 APP_KIND_TABLE = "KINDS__"
 METADATA_TABLE = "METADATA__"
+DATASTORE_VERSION_TABLE = "DATASTORE_VERSION__"
```

Comment:
Can we make this DATASTORE_METADATA_TABLE? It seems more generic and doesn't conflict with anything that's there or that we have in store for the future.